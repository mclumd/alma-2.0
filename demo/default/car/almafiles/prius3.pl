% An uncharged prius with gas, which also has that electric /\ ~charged --> ~powered
% Leads to additional updating because of multiple possible fifs to both charged and uncharged
obs(quote(rel(electric_car, prius))).
obs(quote(rel(gas_tank_car, prius))).
obs(quote(not(rel(charged, prius)))).
obs(quote(rel(fueled, prius))).

fif(and(rel(electric_car, X), not(rel(charged, X))), not(rel(powered, X))).
