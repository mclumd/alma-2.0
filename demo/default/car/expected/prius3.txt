rel(car, prius)
rel(gas_tank_car, prius)
rel(electric_car, prius)
~rel(charged, prius)
rel(fueled, prius)
rel(powered, prius)
abnormal(prius, electric_car, "rel(powered, prius)")
